packages/paste-website/src/pages/theme/changing-theme.mdx

```diff
@@ -62,6 +63,16 @@ Alternatively, Paste will automatically load the default theme's font via JavaSc
 
 **For other themes, see our [manual installation page](/introduction/for-engineers/manual-installation/#how-to-load-the-right-font) for more information.**
 
+## SSR and SSG
+
+If you are using server-side rendering (SSR) or static site generation (SSG), <strong>and</strong> you support theme switching, you will need to use CSS variables and setting a body data-theme attribute instead to avoid theme flicker on initial renders. You can view our <Anchor href="/blog/2025-03-20-css-variables-ssr-ssg/">blog post</Anchor> on this topic, discussing the problem and demonstrating how we solved it in the Paste website. 
```

Comment:
```suggestion
If you are using server-side rendering (SSR) or static site generation (SSG), <strong>and</strong> you support theme switching, you will need to use CSS variables and set a `body` `data-theme` attribute to avoid theme flicker on initial renders. You can view our <Anchor href="/blog/2025-03-20-css-variables-ssr-ssg/">blog post</Anchor> on this topic, discussing the problem and demonstrating how we solved it for the Paste website. 
```